docs/stale-subvolume.md

```diff
@@ -0,0 +1,32 @@
+# stale-subvolume cleanup
```

Comment:
How about this format,

stable-subvolume: which talks about stale volume in general(with main heading `#`)
cleanup: to discuss about cleaning stale volume with examples(with sub-heading `##`)
ls: same as clean-up
delete: same as clean-up